lib/markdown2.py

```diff
@@ -2722,6 +2722,41 @@ def run(self, text):
         return self.admonitions_re.sub(self.sub, text)
 
 
+class Alerts(Extra):
+    '''
+    Markdown Alerts as per
+    https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts
+    '''
+
+    name = 'alerts'
+    order = (), (Stage.BLOCK_QUOTES, )
+
+    alert_re = re.compile(r'''
+        <blockquote>\s*
+        <p>
+        \[!(?P<type>NOTE|TIP|IMPORTANT|WARNING|CAUTION)\]
+        (?P<closing_tag></p>[ \t]*\n?)?
+        (?P<contents>[\s\S]+?)
+        </blockquote>
+    ''', re.X
+    )
+
+    def test(self, text):
+        return "<blockquote>" in text
+
+    def sub(self, match: re.Match) -> str:
+        typ = match["type"].lower()
+        heading = f"<em>{match["type"].title()}</em>"
```

Comment:
Bad f string here? CI failing on it

Reply:
Thanks! I should have tested not only on 3.12 where this works. :)